tests/dialects/test_presto.py

```diff
@@ -177,6 +177,17 @@ def test_regex(self):
                 "spark": "ARRAY_JOIN(x, '-', 'a')",
             },
         )
+        self.validate_all(
+            "STRPOS('ABC', 'A', 3)",
+            read={
+                "trino": "STRPOS('ABC', 'A', 3)",
+            },
+            write={
+                "presto": "STRPOS('ABC', 'A', 3)",
+                "trino": "STRPOS('ABC', 'A', 3)",
+                "snowflake": "POSITION('A', 'ABC')",
```

Comment:
Should we log an "unsupported" warning here? So if `instance` is set and the `generate_instance` is not, then `self.unsupported("Instance parameter is not supported by STRPOS")` or something.

Reply:
In general I'd be a fan of it, but note that not all dialects use `str_to_position_sql`. For example, Snowflake generates `exp.StrPosition` on it's own, so by transpiling Trino/Presto to Snowflake you won't hit the same unsupported warning as you would by transpiling to e.g. Postgres, even though both semantically would have `generate_instance = False`. Also, since having "overloaded" expressions with potentially unnecessary args is a common phenomenon, I thought that silently dropping `instance` is OK in this case. 

In the case that we do want to go forward with an unsupported warning, I've already prepared a commit with the necessary changes

Reply:
That's ok, if there's a split like that in the generator method overrides we don't have to introduce complexity. This is something we could improve in the future.